Makefile

```diff
@@ -43,6 +43,7 @@ export TESTARGS ?= $(TESTARGS_DEFAULT)
 DEST ?= $(GOPATH)/src/$(GIT_HOST)/$(BASE_DIR)
 VERSION ?= $(shell cat COMPONENT_VERSION 2> /dev/null)
 IMAGE_NAME_AND_VERSION ?= $(REGISTRY)/$(IMG)
+CONTROLLER_NAMESPACE ?= open-cluster-management-agent-addon
 # Handle KinD configuration
 KIND_NAME ?= test-managed
 KIND_NAMESPACE ?= open-cluster-management-agent-addon
```

Comment:
To avoid repetition, would you please set `KIND_NAMESPACE` to the new variable also:
```
CONTROLLER_NAMESPACE ?= open-cluster-management-agent-addon
KIND_NAMESPACE ?= $(CONTROLLER_NAMESPACE)
```

Reply:
done as suggested.

Reply:
Thanks!